### ISSUE TITLE
Death Grip not working

### ISSUE THREAD

**Issue by torrenteeselkmanda** [dalaranwow/dalaran-wow#5917]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: Death grip, Death Knight spell won't work on bosses, they are completelly inmune

**Expected behaviour**: Bosses should be inmune to the drag effect but they should recieve a the debuff ID 51399 (https://www.wowhead.com/spell=51399/death-grip)

**Steps to reproduce the problem**:

1. Try to use Death grip while don't have the aggro
2. It will say INMUNE

**Include proofs for this behaviour**
The comments on Glyph of dark Command say, back in the day, is not worth it because you can use Death grip if you miss the taunt
https://www.wowhead.com/item=43538/charred-glyph#comments

You can also find more info in the comments section of Death grip ability on wowhead
https://www.wowhead.com/spell=49576/death-grip

Some notable comments are:

By clovis6780 (6,828 – 2·15·48) on 2010/08/20 (Patch 3.3.5) |   |  
-- | -- | --
While many mobs, especially bosses, are immune to the grip effect but not the taunt effect, it may be possible that some mobs are susceptible to the grip effect but immune to the taunt effect.

I believe I observed it last night, but alas, I wasn't logging combat and I didn't think to check before I logged out.

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**

You can research this on several pages like https://wowgaming.altervista.org/aowow/ or https://wowhead.com/

**Include Screenshots from the issue if necessary**

Death grip spell seems to not working properly on dungeon/raid bosses. Yeah, they should be inmune to be dragged onto the DK but they're also inmune to the effect that forces to attack you for a few seconds. I understand that bosses inmune to taunt mechanics will be inmune to this effect, but it makes no sense for the rest of tauntable bosses
